### PR TITLE
Shopware dev environment

### PR DESCRIPTION
**Pull request by lx-wnk**
Added possibility to include environment specific config files

### CODE REVIEW

File: app/config/config.php [hunk 1]
```diff
@@ -15,36 +15,33 @@
...
-
-return array_replace_recursive($this->loadConfig($this->AppPath() . 'Configs/Default.php'), [
-
-    'db' => [
-        'username' => $db['user'],
-        'password' => $db['pass'],
-        'dbname'   => $db['path'],
-        'host'     => $db['host'],
-    ],
+if ('dev' === getenv('SHOPWARE_ENV')) {
```
Comment (bcremer) [inline app/config/config.php]: The composer project should not dictate any environment specific behavior.
The composer project is a template for own shopware based projects which can define environments as they please.

Comment (lx-wnk) [reply app/config/config.php]: That might be correct but I'd like to define the behaviour in the .env file so I could deploy the same files on different environments.

Maybe the 'dev'-flag is too simple for many configurations but I think there should be a possibility to define the whole config (or the relevant environment parts of it) in an .env-file.

If I'm not getting the point please tell me.

**Comment by lx-wnk** [comment]
@bcremer 
I changed the PR could you review it again?

**Comment by bcremer** [comment]
I'm still not seeing the benefit of forcing any config file organization to users of this repository. As stated before, this repo is a template for projects. If you need env specific config files you are free to adjust the code at will. 

See Also:
- https://github.com/shopware/composer-project/pull/10
- https://github.com/shopware/composer-project/pull/12

I am no longer affiliated with shopware so It's not on me to decide. 
Calling in @soebbing for that.

**Comment by soebbing** [comment]
I am with @bcremer on this one. This repo is a template, feel free to fork and modify it to your need! We try to keep it basic, so it can be a starting point for whatever your shop-project needs to achieve.

**Pull request closed without merging**